recipes-ccsp/ccsp/rdk-wanmanager.bb

```diff
@@ -8,13 +8,10 @@ DEPENDS_append = " ${@bb.utils.contains('DISTRO_FEATURES', 'rdkb_wan_manager', '
 
 require recipes-ccsp/ccsp/ccsp_common.inc
 
-PV = "${RDK_RELEASE}+git${SRCPV}"
-SRC_URI = "git://git@github.com/rdkcentral/RdkWanManager.git;branch=main;protocol=https;name=WanManager"
-SRCREV = "v2.2.0"
-
-
-SRCREV_FORMAT = "WanManager"
 
+SRC_URI = "git://github.com/rdkcentral/RdkWanManager.git;branch=main;name=WanManager"
+SRCREV_WanManager = "v2.2.0"
+PV = "${SRCREV_WanManager}"
```

Comment:
This is failing kirkstone builds:
"
ERROR: rdk-wanmanager-v2.2.0-r0 do_fetch: Bitbake Fetcher Error: FetchError("Recipe uses a floating tag/branch 'v2.2.0' for repo 'github.com/rdkcentral/RdkWanManager.git' without a fixed SRCREV yet doesn't call bb.fetch2.get_srcrev() (use SRCPV in PV for OE).", None)
"
PV has to have SRCPV in it for kirkstone.
Suggest you change it to 
PV = "${SRCREV_WanManager}+git${SRCPV}"
or something like that.

Same is true for the other recipes.